plugins/woocommerce/includes/tracks/class-wc-site-tracking.php

```diff
@@ -83,11 +86,6 @@ public static function add_tracking_function() {
 				const eventName = '<?php echo esc_attr( WC_Tracks::PREFIX ); ?>' + name;
 				let eventProperties = properties || {};
 				eventProperties = { ...eventProperties, ...<?php echo json_encode( $filtered_properties ); ?> };
-				if ( window.wp && window.wp.hooks && window.wp.hooks.applyFilters ) {
-					eventProperties = window.wp.hooks.applyFilters( 'woocommerce_tracks_client_event_properties', eventProperties, eventName );
-					delete( eventProperties._ui );
```

Comment:
as per Slack conversation- these changes come from https://github.com/woocommerce/woocommerce/pull/25851 and as such could be considered for removal.

Reply:
Do you have a link to that Slack thread for context?

I'm not opposed to deleting these, but this is the equivalent of removing the `unset()`s on the backend tracks.  I'm really still unclear on why these are even in the code so I'm not opposed to removing them, but seems like we should remove both or keep both.